docs/get-invited-and-onboard-to-techpass.md

```diff
@@ -38,23 +36,25 @@ You need the following to get invited to TechPass and complete the onboarding:
   >
   > **When you are invited to TechPass**
   >- A TechPass account is provisioned for you and is in pending state.
-  >- We'll send the TechPass onboarding email to activate the account. This email is valid only for 30 days and will have your TechPass account or log in ID. 
-  >- Your TechPass log in ID's domain will be ``techpass.gov.sg```.
+  >- We'll send the TechPass onboarding email with your TechPass account or log in ID. 
+  >- You need to activate the account within 30 days. 
+  >- Your TechPass log in ID's domain will be ```techpass.gov.sg```.
```

Comment:
will be or is?

Reply:
I had previously checked with the TP dev team and got confirmed that it should be domain but I too had the same query :-)